### DISCUSSION TITLE
key press sounf

### DISCUSSION THREAD

**Discussion by benutzer74** [AnySoftKeyboard/AnySoftKeyboard#2637] [Q&A]
Hello! You can enable a key press sound, but where do you set the Volume for this keypress sound? Or is this set in the operating system itself?
And is swyping supported?
Thanks a lot!

OK, I found now the key click volume setting, just wonder about the swype :-)
Guess what, I even found out now, swipe is a beta feature :-)

**Comment by menny** [comment]
It is 😃
